src/cow-react/modules/mainMenu/typings.ts

```diff
@@ -0,0 +1,73 @@
+import { TradeStateFromUrl } from '@cow/modules/trade/types/TradeState'
```

Comment:
Why typings.ts? 

I belive we've called them `types` most of the times
<img width="597" alt="image" src="https://user-images.githubusercontent.com/2352112/199559058-ac7e6d6b-df64-4b8c-bd15-25ea1d9251fc.png">

I can see now, some of the new ones not 
<img width="544" alt="image" src="https://user-images.githubusercontent.com/2352112/199559250-56677ce3-5393-4278-8e31-3a8d6181f5d9.png">

Reply:
Thanks! Renamed them all